src/verbose-operation/verbose-operation-view-styles.ts

```diff
@@ -1,43 +1,55 @@
-import { makeStyles, shorthands } from "@fluentui/react-components";
+import { makeStyles, shorthands, tokens } from "@fluentui/react-components";
 
 export const useStyles = makeStyles({
   operationView: {
     display: "flex",
     flexDirection: "column",
     flexGrow: 2,
-    color: "black",
     minWidth: "15rem",
     height: "100%",
     minHeight: 0,
   },
   operationNameAccPanel: {
     whiteSpace: "pre-wrap",
     marginLeft: "1rem",
-    backgroundColor: "white",
+    ...shorthands.padding("1rem"),
+    ...shorthands.borderRadius("10px"),
+    backgroundColor: tokens.colorNeutralForegroundInverted,
   },
   operationVariablesAccPanel: {
     whiteSpace: "pre-wrap",
     marginLeft: "1rem",
-    backgroundColor: "white",
+    ...shorthands.padding("1rem"),
+    ...shorthands.borderRadius("10px"),
+    backgroundColor: tokens.colorNeutralForegroundInverted,
   },
   durationAccPanel: {
     marginLeft: "1rem",
     display: "flex",
     flexDirection: "column",
-    backgroundColor: "white",
+    ...shorthands.padding("1rem"),
+    ...shorthands.borderRadius("10px"),
+    backgroundColor: tokens.colorNeutralForegroundInverted,
+  },
+  accPanel: {
+    ...shorthands.padding("1rem"),
+    ...shorthands.borderRadius("10px"),
```

Comment:
use rem and instead of adding same styles everywhere, extract out it to common function and call that function from here